### PR TITLE
chore(INJI-241): bump up tuvali version to v0.4.4

### PR DESCRIPTION
**Pull request by adityankannan-tw**
Bumping up tuvali version to 0.4.4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "react-native-simple-markdown": "^1.1.0",
     "react-native-svg": "12.1.1",
     "react-native-swipe-gestures": "^1.0.5",
-    "react-native-tuvali": "github:mosip/tuvali#v0.4.3",
+    "react-native-tuvali": "github:mosip/tuvali#v0.4.4",
```
Comment (vijay151096) [inline package.json:83]: please do run npm i && ios && pod install also along with this

Comment (adityankannan-tw) [reply package.json:83]: Have updated the package-lock.json

File: package-lock.json [hunk 2]
```diff
@@ -45243,32 +45203,7 @@
...
-        "scheduler": {
-          "version": "0.19.1",
-          "resolved": "https://registry.npmjs.org/scheduler/-/scheduler-0.19.1.tgz",
-          "integrity": "sha512-n/zwRWRYSUj0/3g/otKDRPMh6qv2SYMWNq85IEa8iZyAv8od9zDYpGSnpBEjNgcMNq6Scbu5KfIPxNF72R/2EA==",
-          "peer": true,
-          "requires": {
-            "loose-envify": "^1.1.0",
-            "object-assign": "^4.1.1"
-          }
+          "integrity": "sha512-KMvVuFzpKBuiIXW3E4u3mySRO2/mCHSyZDJQM5NQ9Q9KHWHWh0NHgfbRMLLrceUK5qAL4ytALJbpRMjixFZh8A=="
```
Comment (vijay151096) [inline package-lock.json:45206]: Other package changes also got updated.

**Review by vijay151096 — APPROVED** [review]
looks good to me.

**Comment by swatigoel** [comment]
Don't merge it. Decision is pending to merge on release or develop

File: package-lock.json [hunk 3]
```diff
@@ -45633,8 +45633,7 @@
     },
     "react-native-tuvali": {
       "version": "git+ssh://git@github.com/mosip/tuvali.git#768a7fc8b1cd50db97dc5f47d5d70b66ca9d5a3f",
```
Comment (vharsh) [inline package-lock.json:45635]: The checksums for `react-native-tuvali` dependency should have also changed on a version bump up, please run this command as well `npm i react-native-tuvali@github:mosip/tuvali#v0.4.4` and then commit the `package-lock.json` & `Podfile.lock` changes.

Comment (adityankannan-tw) [reply package-lock.json:45635]: Added the missed changes also

**Review by vharsh — APPROVED** [review]
lgtm

**Pull request closed without merging**